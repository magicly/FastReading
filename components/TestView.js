import React from 'react';

import {
  Text,
} from 'react-native';

export default ({ name }) => (
  <Text>text: {name}, time is: {Date.now()}</Text>
);