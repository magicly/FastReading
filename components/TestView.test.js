import React from 'react';
import TestView from './TestView';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  Date.now = jest.fn(() => 1514552641273);
  const tree = renderer
    .create(<TestView name="magicly007" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});