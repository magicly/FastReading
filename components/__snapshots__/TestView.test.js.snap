// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
>
  text: 
  magicly007
  , time is: 
  1514552641273
</Text>
`;
